code/modules/client/preferences.dm

```diff
@@ -472,3 +461,92 @@ datum/preferences
 
 /datum/preferences/proc/close_load_dialog(mob/user)
 	user << browse(null, "window=saves")
+
+// Logs a character to the database. For statistics.
+/datum/preferences/proc/log_character(var/mob/living/carbon/human/H)

```

Comment:
It would be useful to add a new log when the mind.assigned_role changes to be able to get all the antags from a round with a simple query.